code/index.html

```diff
@@ -1,30 +1,324 @@
+            <div class="select-container__text">
+              <span class="signup__options">
+                Financial (50+)
+              </span>
+              </label>
+            </div>
+          </div>
+          <!-- Insurance option -->
+          <div class="select-container">
+            <div class="select-container__object">
+              <label for="insurance" class="checkbox--remove">
+                <input type="checkbox" id="insurance" name="categories" value="insurance" />
+                <span class="checkbox--custom"></span>
+            </div>
+            <div class="select-container__text">
+              <span class="signup__options">
+                Insurance (40+)
+              </span>
+              </label>
+            </div>
+          </div>
+          <!-- Daily Digest option -->
+          <div class="select-container">
+            <div class="select-container__object">
+              <label for="dailyBlank" class="checkbox--remove">
+                <input type="checkbox" id="dailyBlank" name="categories" value="dailyBlank" />
+                <span class="checkbox--custom"></span>
+            </div>
+            <div class="select-container__text">
+              <span class="signup__options">
+                Daily Digests (80+)
+              </span>
+              </label>
+            </div>
+          </div>
+        </div>
+
+        <!-- Accept Terms -->
+        <div class="accept-container">
+          <div class="accept-container__checkbox">
+            <label for="acceptTerms" class="checkbox--remove">
+              <input type="checkbox" id="acceptTerms" name="acceptTerms" value="termsAccepted" required />
+              <span class="checkbox--custom--lg"></span>
+          </div>
+          <div class="accept-container__text">
+            <span class="signup__options">
+              I agree this person is terrible and deserves to be punished.
+            </span>
+            </label>
+          </div>
+        </div>
+
+        <!-- Submit Button -->
+        <button class="btn" type="submit">
+          ✞ SUBMIT ✞
+        </button>
+      
+      <!-- End of Form and Form Container -->
+      </form>
+    </div>
+  <!-- End of Page section -->
+  </section>
+
+  <!-- Footer section (Commented out, I don't want one) -->
+  <!-- <footer class="footer">
+    <p class="text footer__text">
+      © 2022 DEATH BY SPAM
+    </p>
+    <a class="footer__link" href="https://www.flaticon.com/free-icons/">Icons created by Freepik - Flaticon</a>
+  </footer> -->
+
+  <!-- Black Screen of Death -->
+  <div class="black-screen">
+    <div class="black-screen__container">
+      <img class="black-screen__icon" src="images/cementery.png" alt="Cemetary Icon" />
+      <h1 class="title title--large black-screen__text" id="restInPeace">
+        Rest In Peace
+      </h1>
+      <button class="black-screen__button" onclick="history.go(0);">
+        Kill Again?
+      </button>
+    </div>
```

Comment:
This part is just amazing!

Reply:
If you want me to go through it with you and explain what's going on at some point let me know!

I love your daughter's comment 😆